mesonbuild/modules/pkgconfig.py

```diff
@@ -351,7 +351,8 @@ def _generate_pkgconfig_file(self, state, deps, subdirs, name, description,
                 ofile.write('prefix={}\n'.format(self._escape(prefix)))
                 if uninstalled:
                     ofile.write('srcdir={}\n'.format(self._escape(srcdir)))
-                ofile.write('libdir={}\n'.format(self._escape('${prefix}' / libdir)))
+                if len(deps.pub_libs) > 0 or len(deps.priv_libs) > 0:
```

Comment:
These are initialized as empty lists and if they are length 0 then they are going to be `False`'y.

```suggestion
                if deps.pub_libs or deps.priv_libs:
```

Reply:
Much better, thanks :)